independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/JavaVersion.java

```diff
@@ -62,7 +62,7 @@ public String toString() {
     }
 
     // ordering is important here, so let's keep them ordered
-    public static final SortedSet<Integer> JAVA_VERSIONS_LTS = new TreeSet<>(List.of(17, 21));
+    public static final SortedSet<Integer> JAVA_VERSIONS_LTS = new TreeSet<>(List.of(11, 17, 21));
```

Comment:
Wouldn't this mean that the Java 11 will be shown as an option when we release Quarkus 3.8 and it's an LTS, even though Java 17 will be the minimum version?

Reply:
@geoand did you see my comment?

Reply:
I did